scripts/quarantine_windows_mac.yaml

```diff
@@ -8,7 +8,6 @@
 #    - None
 
 - platforms:
-    - native_posix
```

Comment:
Beware that native_posix still exists as a platform in Zephyr. Though it is not anymore a default test platform. 
So this change may or may not be a problem depending on how twister is called (note that leaving native_posix as meant to be excluded should not be a problem)